infrastructure/aws/production/ecs-task-definition-web.json

```
@@ -125,6 +125,12 @@
                     "name": "INFLUXDB_URL",
                     "value": "https://eu-central-1-1.aws.cloud2.influxdata.com"
                 },
+                {
+
+                    "name": "AWS_SSE_LOGS_BUCKET_NAME",
+                    "value": "flagsmith-fastly-logs-production"
+
+                },
```

Weird formatting here?
```suggestion
                {
                    "name": "AWS_SSE_LOGS_BUCKET_NAME",
                    "value": "flagsmith-fastly-logs-production"
                },
```